sim-cli/src/main.rs

```diff
@@ -199,3 +217,47 @@ async fn main() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+async fn read_sim_path(data_dir: PathBuf, sim_file: PathBuf) -> anyhow::Result<PathBuf> {
+    let sim_path = if sim_file.is_relative() {
+        data_dir.join(sim_file)
+    } else {
+        sim_file
+    };
+
+    if sim_path.exists() {
+        Ok(sim_path)
+    } else {
+        log::info!("Simulation file '{}' does not exist", sim_path.display());
+        let sim_files: Vec<String> = std::fs::read_dir(data_dir.clone())?
+            .filter_map(|f| {
+                f.ok().and_then(|f| {
+                    if f.path().extension()?.to_str()? == "json" {
+                        return f.file_name().into_string().ok();
+                    }
+                    None
+                })
+            })
+            .collect::<Vec<_>>();
+
+        if sim_files.is_empty() {
+            anyhow::bail!("no simulation files found in {}.", data_dir.display());
```

Comment:
Given `data_dir` defaults to `.`, this error reads pretty confusingly:

```
Error: no simulation files found in ..
```

I'd drop the trailing `.` or expand the local path to absolute